pycbc/io/hdf.py

```diff
@@ -870,15 +922,16 @@ def to_coinc_xml_object(self, file_name):
             coinc_inspiral_row.coinc_event_id = coinc_id
             coinc_inspiral_row.mchirp = sngl_combined_mchirp
             coinc_inspiral_row.mass = sngl_combined_mtot
-            coinc_inspiral_row.set_end(\
-                                   LIGOTimeGPS(coinc_event_vals['time1'][idx]))
-            coinc_inspiral_row.snr = coinc_event_vals['stat'][idx]
+            coinc_inspiral_row.set_end(
+                LIGOTimeGPS(coinc_event_vals['time'][idx])
+            )
+            coinc_inspiral_row.snr = net_snrsq**0.5
             coinc_inspiral_row.false_alarm_rate = coinc_event_vals['fap'][idx]
             coinc_inspiral_row.combined_far = 1./coinc_event_vals['ifar'][idx]
             # Transform to Hz
             coinc_inspiral_row.combined_far = \
                                     coinc_inspiral_row.combined_far / YRJUL_SI
-            coinc_event_row.likelihood = 0.
+            coinc_event_row.likelihood = coinc_event_vals['stat'][idx]
```

Comment:
I don't know if setting the statistic to be the 'likelihood' is the right thing to do.
Though compared to calling it the network SNR it probably is more accurate.

Reply:
I'm going to insist on this one. We can *not* any more put the statistic in the SNR column now that our statistic is nothing like SNR. It will confuse people considerably if graceDB reports SNRs that are wildly different from the actual SNRs!

The only column then remaining is likelihood, and this does appear on graceDB. So this is now officially our "stat" column (we should tell graceDB that in a code edit before O4!)

Reply:
I was also happy with the "SNR" not being the statistic - if gracedb will report this as something that isn't 'likelihood' then this is fine. My issue was with calling something that isn't the likelihood the likelihood. So long as that doesn't get passed on this is fine